packages/next/taskfile.js

```diff
@@ -623,6 +623,15 @@ export async function ncc_browserslist(task, opts) {
   await task
     .source(relative(__dirname, require.resolve('browserslist')))
     .ncc({ packageName: 'browserslist', externals })
+    // eslint-disable-next-line require-yield
+    .run({ every: true }, function* (file) {
+      const source = file.data.toString()
+      // We replace the module/chunk loading code with our own implementation in Next.js.
+      file.data = source.replace(
```

Comment:
![modern-problems-funny-gif-gif-by-moodman](https://github.com/user-attachments/assets/46a929c9-c941-443a-be1f-47d683d0327c)

Reply:
If we make ncc support customized env later this would be easier